app/src/main/java/org/oppia/app/application/OppiaApplication.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.app.application
+
+import android.app.Application
+import android.content.Context
+
+/** Oppia MainApplication class for global applicationContext
+ * Called when the application is starting, before any activity, service, or receiver objects (excluding content providers) have been created.
+ */
+class OppiaApplication : Application() {
```

Comment:
We can avoid exposing the application instance in a singleton by using dependency injection, instead. Exposing the application in this way is an anti-pattern, and it makes properly switching the test context challenging. See my comment below for converting Logger to a injectable class.

Reply:
done